src/index.ts

```diff
@@ -195,9 +196,10 @@ async function main() {
   } else {
     processedBlock = newprocessedBlock;
   }
+  ethPrice = await getETHPriceInUSD();
   await inspectBlockBuys(latestBlock, BigInt(TOKEN_ADDRESS));
 }
 
-cron.schedule("*/30 * * * * *", main);
+cron.schedule("*/1 * * * * *", main);
```

Comment:
Not a great idea imo as blocktime is something around 30secs, so anything lower than 10s is completely overkill imo

Reply:
So, let's use 10s because with 30 we can miss txs